src/js/sdk.ts

```diff
@@ -71,10 +71,10 @@ export function init(
         },
       })
     );
+    options = assignDefaultOptions(options);
```

Comment:
```suggestion
    options = {
        ...DEFAULT_OPTIONS,
        ...options
    }
```
Instead of using a function call, I would do this and define the `DEFAULT_OPTIONS` as a constant object.

Reply:
This file needs a massive refactor/clean now that I look at it, but I'll take care it later.

Reply:
good idea 👍 
addressed in the [most recent commit](https://github.com/getsentry/sentry-react-native/pull/1007/commits/d6e81a5607fdf29f66921d5ec69a4ee4a3f58fc0)

Reply:
Oh the build is failing still, this would be fixed if you move the default options setting code before the if statement that sets the default integrations above it. Otherwise everything else looks good.

Reply:
gotchya, fixed that & the build is passing now